certified-connectors/Scancloud/apiDefinition.swagger.json

```diff
@@ -338,7 +338,7 @@
             }
           }
         },
-        "summary": "Scan Emails using SecPlugs Scan",
+        "summary": "Scan Emails using Scancloud",
         "operationId": "emailscan",
```

Comment:
@Ajaatsecplugs please [use pascal casing for operationId](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#actions-and-triggers) properties.